spec/requests/providers_spec.rb

```diff
@@ -342,8 +345,13 @@
           end
         end
 
+        let!(:json) { Map JSON.parse response.body }
+
         it 'responds with bad request' do
+          # rubocop:disable Metrics/LineLength
```

Comment:
If this rubocop rule is too onerous, I'd rather take it out than one-off disable it.

Reply:
I disabled this check and removed other references (primarily in tests)